python/hsml/sklearn/predictor_config.py

```diff
@@ -0,0 +1,38 @@
+#
+#   Copyright 2022 Logical Clocks AB
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+from hsml.predictor_config import PredictorConfig
+
+
+class PredictorConfig(PredictorConfig):
+    """Configuration for a predictor running a sklearn model."""
+
+    def __init__(
+        self,
+        serving_tool="DEFAULT",
```

Comment:
Could use the class PREDICTOR constants from constants.py instead for these files.

Reply:
Done